src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-usersgroups-list/smb-usersgroups-list.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-container *ngIf="usersGroups$ | async as usersGroups">
+  <cd-table
+    [data]="usersGroups"
+    columnMode="flex"
+    [columns]="columns"
+    selectionType="single"
+    [hasDetails]="true"
+    (setExpandedRow)="setExpandedRow($event)"
+    (fetchData)="loadUsersGroups()"
+>
+  <cd-smb-usersgroups-details
+    *cdTableDetail
+    [selection]="expandedRow"
+  ></cd-smb-usersgroups-details>
+</cd-table>
+</ng-container>
+
+<ng-template
+  #groupsNamesTpl
```

Comment:
nit
```suggestion
  #groupNamesTpl
```

Reply:
Actually, I've been following the naming of the backend variables, and this is for the `SMB Usersgroups` both users and groups as plural since this resource can have multiple users and multiple groups. 
So leaving it is as it is.

Ref: https://docs.ceph.com/en/latest/mgr/smb/#users-and-groups-resource

Reply:
alright, thanks